template_qubuntu/02_install_groups_post.sh

```diff
@@ -4,7 +4,7 @@
 source "${SCRIPTSDIR}/vars.sh"
 source "${SCRIPTSDIR}/distribution.sh"
 
-if  [ $DIST = "trusty" ]; then
+if  [ $DIST != "trusty" ]; then
```

Comment:
This looks wrong - according to your comment, trusty build _do_ need this section.
BTW Why is it needed in the first place? Something about pulseaudio version in trusty (which one is there?), or not having pulseaudio at all?